src/libraries/Native/Unix/System.Security.Cryptography.Native.Apple/pal_ssl.c

```diff
@@ -594,7 +594,7 @@ int32_t AppleCryptoNative_SslSetEnabledCipherSuites(SSLContextRef sslContext, co
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
         // macOS
-        return SSLSetEnabledCiphers(sslContext, cipherSuites, (size_t)numCipherSuites);
+        return SSLSetEnabledCiphers(sslContext, (const SSLCipherSuite *)cipherSuites, (size_t)numCipherSuites);
```

Comment:
The warning you have shared says that `SSLCipherSuite` is `const unsigned short`. Is this cast safe? Maybe SSLCipherSuite is defined as 32-bit uint on x64 and 16-bit uint on arm64?

Reply:
Ah, I see - this code-path is only taken when SSLCipherSuite is uint32_t.

Reply:
We should be able to delete `#if !defined(TARGET_IOS) && !defined(TARGET_TVOS)` around this block with this fix.